cmd/sdkstats/main.go

```diff
@@ -7,46 +7,141 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"strings"
 
 	"github.com/go-github-metrics-1/pkg/sdkstats"
```

Comment:
This line needs to match the actual repo path:
```suggestion
	"github.com/learnoperators/go-github-metrics/pkg/sdkstats"
```

Reply:
I am guessing I cant refer to this until I merge DEV with MASTER on learnoperators/go-github-metrics

Reply:
I think you can. In your local environment, you can get this working in one of two ways:
1. Move the project directory into `$GOPATH/src/github.com/learnoperators/go-github-metrics`
2. Create `go.mod` file in the project root with `go mod init github.com/learnoperators/go-github-metrics`

I suggest option 2 since the SDK uses [Go Modules](https://github.com/golang/go/wiki/Modules) and it is now the official dependency manager supported by Go's built-in tooling.

Reply:
Done